app/javascript/components/provider-dashboard-charts/charts_config.js

```diff
@@ -1,8 +1,11 @@
 import moment from 'moment';
 
-const dailyTimeTooltip = () => (data) => {
+const dailyTimeTooltip = (units = undefined) => (data) => {
   const theMoment = moment(data[0].date);
-  const htmlString = `<div class="tooltip-inner">${theMoment.format('MM/DD/YYYY')}  ${data[0].value} ${data[0].group} </div>`;
+  let htmlString = `<div class="tooltip-inner">${theMoment.format('MM/DD/YYYY')}  ${data[0].value} ${data[0].group} </div>`;
+  if (units) {
+    htmlString = `<div class="tooltip-inner">${theMoment.format('MM/DD/YYYY')}  ${data[0].value} ${__(units)} </div>`;
+  }
```

Comment:
I'm fine with this, but we should probably look for a localized way to present the date instead.

Reply:
For example, you can use `L` here instead of `MM/DD/YYYY`, which will be the same thing for the en-us locale, but will change in other locales.

Reply:
@GilbertCherrie Thoughts on this?  On re-review, I think this might be a better approach even if we don't immediately localize it

Reply:
Yeah I can localize it using your suggested method